### PR TITLE
Adds readiness and liveness probe in db deployment

### PR DESCRIPTION
**Pull request by SM43**
# Changes
Adds readiness and liveness probe in db deployment

Signed-off-by: Shivam Mukhade <smukhade@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: api/config/12-db-deployment.yaml [hunk 1]
```diff
@@ -57,6 +57,16 @@ spec:
           volumeMounts:
             - name: db
               mountPath: /var/lib/postgresql/data
+          readinessProbe:
+            exec:
+              command: [psql, -w, -U, postgres, -d, $(POSTGRES_DB), -c, "SELECT 1"]
```
Comment (sthaha) [inline api/config/12-db-deployment.yaml:62]: does this work? what does $(POSTGRES_DB) eval to?

Comment (sthaha) [inline api/config/12-db-deployment.yaml:62]: if the variable interpolation doesn't work, how about we use `bash -c psql .... ` ?

Comment (SM43) [reply api/config/12-db-deployment.yaml:62]: `$(POSTGRES_DB)` refers value from env variable defined above

Comment (sthaha) [reply api/config/12-db-deployment.yaml:62]:
```  
┆ ┆ ┆ ┆ ┆ - bash
  ┆ ┆ ┆ ┆ ┆ - -c
  ┆ ┆ ┆ ┆ ┆ - psql -w -U ${POSTGRES_USER} -d ${POSTGRES_DB} -c 'select 1 as query;'
  ```
works with env

Comment (SM43) [reply api/config/12-db-deployment.yaml]: It works. Thank you :)

**Pull request merged**